source/TradingPanel.cpp

```diff
@@ -123,31 +129,21 @@ void TradingPanel::Draw()
 	font.Draw("free:", Point(MIN_X + SELL_X + 5, lastY), selected);
 	font.Draw(to_string(player.Cargo().Free()), Point(MIN_X + HOLD_X, lastY), selected);
 
-	int outfits = player.Cargo().OutfitsSize();
 	int missionCargo = player.Cargo().MissionCargoSize();
-	sellOutfits = false;
-	if(player.Cargo().HasOutfits() || missionCargo)
+	double minableCargo = player.Cargo().MinablesSizePrecise();
+	double outfitCargo = max(0., player.Cargo().OutfitsSizePrecise() - minableCargo);
```

Comment:
If you want to avoid floating-point errors, maybe just make a Cargo member function that calculates only non-minables.